notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>I love this plot! I don't know why it makes me so happy. But I worry about its accessibility by relying too much on colors. I probably would struggle to pick out the difference between HLA or HLSP or HST on the plot from the legend, and I don't have any issues with viewing color. That said, there are a lot of symbols and you don't want it to get too messy! In addition to the color, could we also cycle through symbols, like triangle, circle, square? And then filled or not filled? Then by the time you loop through those 6 different types you're at a sufficiently different color that it should be okay. But again, if that looks too chaotic and messy, we can figure our something else. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>